src/main/features/offer/routes/agreement.ts

```diff
@@ -0,0 +1,41 @@
+import * as express from 'express'
+import { Paths } from 'offer/paths'
+import { ErrorHandling } from 'common/errorHandling'
+import { DocumentsClient } from 'app/documents/documentsClient'
+import * as http from 'http'
+import * as HttpStatus from 'http-status-codes'
+import { User } from 'idam/user'
+
+const documentsClient: DocumentsClient = new DocumentsClient()
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.agreementReceiver.uri,
+    ErrorHandling.apply(
+      async (req: express.Request, res: express.Response, next: express.NextFunction) => {
+        const user: User = res.locals.user
+
+        documentsClient.getSettlementAgreementCopy(user.claim.externalId)
+          .on('response', (response: http.IncomingMessage) => {
```

Comment:
I believe the download logic is just copied over, would be nice to have a component for it.